extras/catch_amalgamated.hpp

```diff
@@ -1129,6 +1129,7 @@ namespace Catch {
 
         //! Deriving classes become noncopyable and nonmovable
         class NonCopyable {
+        public:
```

Comment:
https://github.com/catchorg/Catch2/blob/devel/docs/contributing.md#using-github

> When submitting a pull request please do not include changes to the amalgamated distribution files. This means do not include them in your git commits!

The amalgamated header and source file get updated with each new release. They are not meant to be edited manually.

Reply:
Thank you for your review. I reverted the change in amalgam.